### DISCUSSION TITLE
Code contribution

### DISCUSSION THREAD

**Discussion by brine0m** [ATVWGS/tenant-management-framework#18] [Q&A] [answered]
Hello, @ATVWGS I and my team would happy to help with code contribution but would like to know the preferred approach for doing this.

Assuming you wouldn't want a merge request to the main branch. Any guidance you can give?

**Comment by ATVWGS** [comment, marked as answer]
Hi @brine0m, if you want to contribute code just create a pull request to the latest prepare-release branch. We will review the request manually and if everything's fine, we will integrate the changes in a future release.
